lib/web/machineid.go

```diff
@@ -55,6 +61,9 @@ func (h *Handler) createBot(w http.ResponseWriter, r *http.Request, p httprouter
 		Bot: &machineidv1.Bot{
 			Metadata: &headerv1.Metadata{
 				Name: req.BotName,
+				Labels: map[string]string{
+					webUIFlowLabelKey: webUIFlowBotGitHubActions,
```

Comment:
I'm a little cautious of this being here when this is a generalised endpoint. We could either have flow specific endpoints, or have the client specify this. I just worry that someone adding another workflow may miss that they need to change this and we'd end up misattributing created resources.

Reply:
I've addressed this along with https://github.com/gravitational/teleport/pull/37348#discussion_r1469755823

Clients will provide this label's key.